README.md

```diff
@@ -7,6 +7,45 @@ PyPA [warehouse](https://github.com/pypa/warehouse) as I feel they have
 made an amazing developer onboarding experience for a very complicated
 project.
 
+## Quickstart
+
+1. Install cookiecutter
+```console
+  $ python3 -m pip install --user cookiecutter
+```
+2. Install kubectl and kind: https://kubernetes.io/docs/tasks/tools/
+
+3. Install Tilt: https://docs.tilt.dev/install.html
+
+4. Create a new project (directly from github):
+```console
+  $ cookiecutter https://github.com/sixfeetup/cookiecutter-sixiedjango/
```

Comment:
On a fresh python virtualenv, this exits with errors:
```
black: command not found

The `black' command exists in these Python versions:
  3.11.2

Note: See 'pyenv help global' for tips on allowing both
      python2 and python3 to be found.
Traceback (most recent call last):
  File "/tmp/tmp1woxicfv.py", line 255, in <module>
    main()
  File "/tmp/tmp1woxicfv.py", line 249, in main
    subprocess.run(shlex.split("isort --profile=black ./backend"))
  File "/home/samuel/.pyenv/versions/3.10.12/lib/python3.10/subprocess.py", line 503, in run
    with Popen(*popenargs, **kwargs) as process:
  File "/home/samuel/.pyenv/versions/3.10.12/lib/python3.10/subprocess.py", line 971, in __init__
    self._execute_child(args, executable, preexec_fn, close_fds,
  File "/home/samuel/.pyenv/versions/3.10.12/lib/python3.10/subprocess.py", line 1863, in _execute_child
    raise child_exception_type(errno_num, err_msg, err_filename)
FileNotFoundError: [Errno 2] No such file or directory: 'isort'
ERROR: Stopping generation because post_gen_project hook script didn't exit successfully
Hook script failed (exit status: 1)
```
This is because I generally don't install anything outside of a virtualenv.
If the packages, `black` and `isort` are installed with `pip install black isort`, then the command works as expected. 

 We could add 
```
$ pip install black isort
```
just above this line, however I think the better plan would be adding them to the requirements during setup, since installation fails without them.